tests/cases/fourslash/codeFixCannotFindModule_suggestion_falsePositive.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path='fourslash.ts' />
+
+// @moduleResolution: node
+// @resolveJsonModule: true
+// @strict: true
+
+// @Filename: /node_modules/foo/bar.json
+////export const x = 0;
+
+// @Filename: /a.ts
+////import abs = require([|"foo/bar.json"|]);
+////abs;
+
+verify.noErrors();
```

Comment:
How come? There should be error in `node_modules/foo/bar.json`  because this is not json?